changelogs/fragments/1833-zfs-creation-only-properties.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - zfs - some zfs properties could be passed when the dataset/volume did not exist, but would fail if the dataset already existed, even if the property matched what was specified in the ansible task
```

Comment:
```suggestion
  - zfs - some ZFS properties could be passed when the dataset/volume did not exist, but would fail if the dataset already existed, even if the property matched what was specified in the ansible task (https://github.com/ansible-collections/community.general/issues/868, https://github.com/ansible-collections/community.general/pull/1833).
```